src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -474,8 +474,12 @@ Status GcsActorManager::CreateActor(const ray::rpc::CreateActorRequest &request,
   auto actor_id = ActorID::FromBinary(actor_creation_task_spec.actor_id());
 
   auto iter = registered_actors_.find(actor_id);
-  if (iter != registered_actors_.end() &&
-      iter->second->GetState() == rpc::ActorTableData::ALIVE) {
+  if (iter == registered_actors_.end()) {
+    RAY_LOG(WARNING) << "Actor " << actor_id << " may be already destroyed.";
```

Comment:
I don't think we need warning logs? When is the case this state is triggered?

Reply:
For example: The owner actor is dead, we will destroy child actor. When we create child actor, will trigger this case.